lib/actions/apiV2.js

```diff
@@ -496,20 +503,19 @@ export function findRoute(params) {
                 stops: patternStops
               }
             })
-            route.patterns = routePatterns
+            newRoute.patterns = routePatterns
             // TODO: avoid explicit behavior shift like this
```

Comment:
Can this comment be removed now?

Reply:
I wish... the problem for me is the `v2` field which I don't like. I think ideally the rest of OTP shouldn't know if it's dealing with OTP1 or OTP2